app/src/main/java/org/listenbrainz/android/model/AppNavigationItem.kt

```diff
@@ -4,7 +4,7 @@ import androidx.annotation.DrawableRes
 import org.listenbrainz.android.R
 
 sealed class AppNavigationItem(val route: String, @DrawableRes val iconUnselected: Int, @DrawableRes val iconSelected: Int, val title: String) {
-    object BrainzPlayer : AppNavigationItem("brainzplayer", R.drawable.player_unselected, R.drawable.player_selected, "Player")
+    object BrainzPlayer : AppNavigationItem("brainzplayer", R.drawable.player_unselected, R.drawable.player_selected, "Local Player")
```

Comment:
Let's not call this a Local Player and stick to Player because the former is taking 2 lines, which is bad

Reply:
In the design file, it was renamed to local player, hence I put that.. it was fine on both the devices I tested (Pixel XL , Oneplus Nord 2). I'll replace it with player for now if it is taking two lines in devices with smaller width.

Reply:
Cool